core/changestream/change.go

```diff
@@ -26,6 +26,10 @@ const (
 	delete
 )
 
+// Created returns if the underlying type has been created. This will encompass
+// if a row has been created in the database.
+const Created = create
```

Comment:
We don't want this. It was removed a few weeks ago.

See https://github.com/juju/juju/pull/18787